docs/python.rst

```diff
@@ -111,7 +111,7 @@ Memory-mapped Registers
 -----------------------
 
 If we look at the generated Fomu header files (to be found for instance in
-`riscv-blink <_static/riscv-blink/include/generated/csr.h>`__), we can see
+:repo:`riscv-blink <riscv-blink/include/generated/csr.h>`__), we can see
```

Comment:
It's not an external link, hence `__` needs to be removed.

Reply:
updated